### DISCUSSION TITLE
Alpha Activate on an image without alpha channel creates blank image

### DISCUSSION THREAD

**Discussion by philipmvp** [ImageMagick/ImageMagick#5602] [Help]
Executing below command creates a blank output image.
`magick "D:\TestDocs\ImageMagick\Input\moon_without_alpha.png" -alpha activate "D:\TestDocs\ImageMagick\Input\moon_alpha_activate.png" `
I am using ImageMagick 7.1.0-45 Q8 x64. The output has a blank alpha channel. If I disable alpha channel from GIMP I can see the image.

**Comment by fmw42** [comment]
Use -alpha set to enable a new alpha channel.  Alpha activate is used generally after alpha deactivate to re-enable a turned off alpha
